### ISSUE TITLE
Manage torrent using Storm

### ISSUE THREAD

**Issue by antoine2tt** [relvacode/storm#27]
Hi !

It's me again, i have succed with the binaries ;)

Is there a way to manager the torrent ? I mean the files inside the torrent ? 

For exemple, i have xx.torrent which contain 2 folders. 

Only one have been downloaded, i want the other one to download, can i do that with storm ? i didn't find a way to...

**Comment by 2600box** [comment]
Agreed, this is needed when adding a torrent and to manage existing ones.
